tests/code_update.py

```diff
@@ -52,6 +57,126 @@ def test_verify_quotes(network, args):
     return network
 
 
+@reqs.description("Test that the security policies table is correctly populated")
+@reqs.snp_only()
+def test_security_policy_table(network, args):
+
+    primary, _ = network.find_nodes()
+    with primary.client() as client:
+        r = client.get("/gov/security_policy")
+        policies = sorted(r.body.json()["policies"], key=lambda x: x["digest"])
+
+    expected = [
+        {
+            "digest": DEFAULT_SNP_SECURITY_POLICY_DIGEST,
+            "raw": DEFAULT_SNP_SECURITY_POLICY,
+        }
+    ]
+    expected.sort(key=lambda x: x["digest"])
+
+    assert policies == expected, [(a, b) for a, b in zip(policies, expected)]
+
+
+@reqs.description(
+    "Node with no security policy set but good digest matching ledger joins successfully"
```

Comment:
I think that here and elsewhere in this file, we should avoid saying "matching ledger" but use un/authorised instead as this test doesn't actually check the ledger files.

Reply:
The tricky part with authorised/unauthorised is either way the policy is authorised (since we authorise the digest). I've had a go at phrasing it better